.gitignore

```diff
@@ -6,9 +6,9 @@ doc/latex
 *.sublime-workspace
 .*.sw*
 .*un~
-build*
+build/*
```

Comment:
I hope I'm not breaking something here. But note that the original statement ignored files such as `.github/workflows/build-linux-wheel.yml`, which I guess is not desirable (at least not for me :) )